pkg/apis/perspective/extension.go

```diff
@@ -152,6 +153,30 @@ func (h Handler) CollectionRouteGetFieldValues(
 		return pvalues, len(pvalues), nil
 	}
 
+	/* Get values of given field name by the following sql:
+	SELECT
+	  DISTINCT COALESCE(namespace, '')
+	FROM
+	  "cost_reports"
+	WHERE
+	  "start_time" >= '2023-09-09T00:00:00Z'
+	  AND "end_time" <= '2023-09-15T16:16:51Z'
+	*/
+
+	fieldStr = fmt.Sprintf(`COALESCE(%s, '')`, req.FieldName)
```

Comment:
what will happened if the req.FieldName is not the column name of table `cost_reports`

Reply:
The view `Validate` already checks the fieldName should be the builtin filter fields